packages/common/driver-definitions/src/storage.ts

```diff
@@ -349,6 +350,11 @@ export interface IDocumentServiceFactory {
 	 */
 	protocolName: string;
 
+	/**
+	 * Factory options which are going to be used to set different features based on factory.
+	 */
+	driverFactoryOptions?: IDriverFactoryOptions;
```

Comment:
I'm not sure why you are adding it here.
Why not reuse existing IDocumentService.IDocumentServicePolicies ?

Reply:
I think we always provide protocol tree on container creation (that would be the only place where we deal with service factory, not service instance). For everything else (normal summaries), I believe we have a service instance, and it can provide us that hint